backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationInitializingService.java

```diff
@@ -126,18 +127,26 @@ public void initAll() {
     // users are in the org.
     List<DemoUser> users = _demoUserConfiguration.getAllUsers();
     for (DemoUser user : users) {
-      OrganizationRole role =
-          user.getAuthorization().getEffectiveRole().orElse(OrganizationRole.getDefault());
-      Organization org =
-          _orgRepo
-              .findByExternalId(user.getAuthorization().getOrganizationExternalId())
-              .orElseThrow(MisconfiguredUserException::new);
       IdentityAttributes identity = user.getIdentity();
       Optional<ApiUser> userProbe = _apiUserRepo.findByLoginEmail(identity.getUsername());
       if (!userProbe.isPresent()) {
         _apiUserRepo.save(new ApiUser(identity.getUsername(), identity));
       }
-      initOktaUser(identity, org, role);
+      OrganizationRoleClaims authorization = user.getAuthorization();
+      if (authorization != null) {
+        OrganizationRole role =
+            authorization.getEffectiveRole().orElse(OrganizationRole.getDefault());
+        LOG.info(
+            "User {} will have role {} in organization {}",
```

Comment:
Huh. That's not actually a pattern I've used in previous splunked-up environments, but I can see where it would be handy if applied with some kind of consistency. (That last part is where I have previously given up and relied on my experience with writing regular expressions...)

Reply:
That said, I don't think tweaking log format strings fits into this week's workload, so I will have to bear it in mind for _future_ logging. 😕